modules/ocf/manifests/packages/powershell/apt.pp

```diff
@@ -0,0 +1,15 @@
+# Include PowerShell apt repo
+class ocf::packages::powershell::apt {
+  apt::key { 'powershell repo key':
+    id     => 'BC528686B50D79E339D3721CEB3E94ADBE1229CF',
+    source => 'https://packages.microsoft.com/keys/microsoft.asc';
+  }
+
+  apt::source { 'powershell':
+    architecture => 'amd64',
+    location     => '[arch=amd64,arm64,armhf] https://packages.microsoft.com/debian/10/prod',
+    release      => 'buster',
```

Comment:
hmm, we still have a number of machines on stretch. consider gating this like https://github.com/ocf/puppet/blob/3567ba2e0dfe17535ab1c5e11ce7f39873150173/modules/ocf_irc/manifests/ircd.pp#L22 ?

Reply:
this supports stretch but there doesn't seem to be a variable that goes:
buster => 10
stretch => 9

I guess I can use an if statement again?

Reply:
I guess if it works it works

Reply:
Like the `lsbmajdistrelease` fact, or?

Reply:
turns out that won't work anyways, because the repo url for stretch is just entirely different

they changed it for buster :P